packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -156,45 +159,113 @@ class AmplifyDataStorePluginTest {
-                ArgumentMatchers.any<Consumer<DataStoreException>>())
+                                                    any<Consumer<DataStoreItemChange<SerializedModel>>>(),
+                                                    any<Consumer<DataStoreException>>())
 
         flutterPlugin.onDelete(mockResult,
-                readMapFromFile("delete_api",
-                        "request/instance_no_predicate.json",
-                        HashMap::class.java) as HashMap<String, Any>)
+                               readMapFromFile("delete_api",
+                                               "request/instance_no_predicate.json",
+                                               HashMap::class.java) as HashMap<String, Any>)
 
         verify(mockResult, times(1)).error(
                 matches("AmplifyException"),
                 matches(FlutterDataStoreFailureMessage.AMPLIFY_DATASTORE_DELETE_FAILED.toString()),
-               // TODO: Accurate match on errorMap
+                // TODO: Accurate match on errorMap
                 any()
-            )
+        )
+    }
+
+    @Test
+    fun test_observe_success_event() {
+
+        flutterPlugin = AmplifyDataStorePlugin(eventHandler = mockStreamHandler)
+        var eventData: HashMap<String, Any> = (readMapFromFile("observe_api",
+                                                               "post_type_success_event.json",
+                                                               HashMap::class.java) as HashMap<String, Any>)
+        var modelData = mapOf("id" to "43036c6b-8044-4309-bddc-262b6c686026",
+                              "title" to "Title 2",
+                              "created" to Temporal.DateTime("2020-02-20T20:20:20-08:00"))
+        var instance = SerializedModel.builder()
+                .serializedData(modelData)
+                .modelSchema(modelSchema)
+                .build()
+
+        var dataStoreItemChange = DataStoreItemChange.builder<SerializedModel>()
+                .item(instance)
+                .initiator(DataStoreItemChange.Initiator.LOCAL)
+                .itemClass(SerializedModel::class.java)
+                .type(DataStoreItemChange.Type.CREATE)
+                .randomUuid()
+                .build()
+
+        doAnswer { invocation: InvocationOnMock ->
+            (invocation.arguments[1] as Consumer<DataStoreItemChange<SerializedModel>>).accept(
+                    dataStoreItemChange)
+            null
+        }.`when`(mockAmplifyDataStorePlugin).observe(any<Consumer<Cancelable>>(),
+                                                     any<Consumer<DataStoreItemChange<out Model>>>(),
+                                                     any<Consumer<DataStoreException>>(),
+                                                     any<Action>())
+
+        flutterPlugin.onSetupObserve(mockResult)
+
+        verify(mockResult, times(1)).success(true)
+        verify(mockStreamHandler, times(1)).sendEvent(eventData)
```

Comment:
Per [Mockito docs](https://javadoc.io/static/org.mockito/mockito-core/3.6.0/org/mockito/Mockito.html#4):
```java
 //following two verifications work exactly the same - times(1) is used by default
 verify(mockedList).add("once");
 verify(mockedList, times(1)).add("once");
```
(That is, you don't _have to_ include the `times(1)`, unless it helps to explicitly document the thing you're testing.)

Reply:
I think it does, to know explicit expectation while reading the test